core/util/compact_merkle_tree_test.go

```diff
@@ -18,7 +18,7 @@ func TestCompactMerkleTreeWithEvenLeaves(t *testing.T) {
 					    [2+3]─────┘               │
 					  3──┘                        │
 				                    [[[0+1]+[2+3]]+[[4+5]+[4+5]]]
-				  	  4──┐                        │
+				  	4──┐                        │
```

Comment:
I think some typing mistake

Reply:
see detail on https://github.com/0chain/blobber/wiki/Protocols#what-is-compactmerkletree

Reply:
one space removed because of which it is not proper tree. this change was not required i think.